src/python/pants/engine/rules_test.py

```diff
@@ -419,11 +419,11 @@ class TestRuleGraph:
     def test_ruleset_with_ambiguity(self) -> None:
         @rule
         def a_from_b_and_c(b: B, c: C) -> A:
-            pass
+            return A()
```

Comment:
Ah.. is this due to a mypy upgrade perhaps..! 💡

Reply:
Yep, mypy is correctly cranky about this now!